src/components/MultiSelect/MultiSelect.tsx

```diff
@@ -22,6 +22,11 @@ export type SelectedParamsType<T> = MultiSelectSelectionProps<T>;
 
 export interface MultiSelectProps<T>
     extends Pick<BlueprintMultiSelectProps<T>, "items" | "placeholder" | "openOnKeyDown"> {
+    /**
+     * Predefined selected values
+     */
+
+    selectedItems?: T[];
```

Comment:
Is there a reason why not to use `selectedItems` from `BlueprintMultiSelectProps<T>` by adding it to the list of picked properties?

Reply:
`BlueprintMultiSelectProps<T>` has `selectedItems` property as required, our component has to be backward compatible. Otherwise each used component needs to be fixed.